lib/colors.py

```diff
@@ -70,8 +70,18 @@ def find_max_value(hist):
 
     return color_index
 
-#def find_color_range(image):
-#    """Returns array """
-#    for hist_color in image_colors(image):
+def find_color_range(image):
+    """Returns array """
```

Comment:
I think we may want to be more specific on this function doc. What are the details of the array are we returning?